backend/libraries/chat_events/src/chat_event_internal.rs

```diff
@@ -213,6 +213,42 @@ impl ChatEventInternal {
             ChatEventInternal::Empty => None,
         }
     }
+
+    pub fn initiated_by(&self) -> Option<UserId> {
+        match self {
```

Comment:
Maybe combine all of the None cases into a single branch

Reply:
I think it's ok as is - the variants are in the same order as they are defined which has some merit